nc-games/src/components/AllReviews/AllReviews.jsx

```diff
@@ -10,17 +10,25 @@ export const AllReviews = () => {
   const [reviews, setReviews] = useState([]);
   const [theCategory, setTheCategory] = useState(undefined);
   const [loading, setLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
 
   useEffect(() => {
     setLoading(true);
-    getReviews(theCategory).then((ApiReviews) => {
-      setLoading(false);
-      setReviews(ApiReviews.reviews);
-    });
+    getReviews(theCategory)
+      .then((ApiReviews) => {
+        setIsError(true);
+        setLoading(false);
+        setReviews(ApiReviews.reviews);
+      })
+      .catch((error) => {
+        setIsError(true);
+      });
   }, [theCategory]);
 
   return loading ? (
     <p>...Loading please wait</p>
+  ) : isError ? (
+    <p>Oops... something went wrong.</p>
```

Comment:
for now this is good enough (handling all errors as the same thing).  later on if you want to add more, you could handle different errors differently (depending on what the status code/msg is on the error object).

Reply:
DONE